py-tensorrt-text-detection/README.md

```diff
@@ -0,0 +1,112 @@
+# Python TensorRT inference example
+
+- [Python TensorRT inference example](#python-tensorrt-inference-example)
+  - [Pre-requisite](#pre-requisite)
+  - [Tested OS Environment](#tested-os-environment)
+  - [Development Environment](#development-environment)
+    - [Install TensorRT](#install-tensorrt)
+      - [Method 1: Native Install](#method-1-native-install)
+      - [Method 2 (Recommended): Use tensorRT container](#method-2-recommended-use-tensorrt-container)
+    - [Install Dependencies](#install-dependencies)
+  - [Model Conversion (ONNX to TensorRT)](#model-conversion-onnx-to-tensorrt)
+    - [Method 1: Using `polygraphy` or `trexec`](#method-1-using-polygraphy-or-trexec)
+    - [Method 2: Using tensorRT libraries (see utility file `trt_utils.py`)](#method-2-using-tensorrt-libraries-see-utility-file-trt_utilspy)
+  - [Related Files for this folder](#related-files-for-this-folder)
+  - [Tips and Questions When Running TensorRT](#tips-and-questions-when-running-tensorrt)
+
+## Pre-requisite
+- You MUST have Nvidia GPU on your device, regardless you are using a physical computer or a virtual machine.
+- Make sure you have cuDNN (matches your cuda version) and cuda installed (check nvcc --version)
+
+## Tested OS Environment
+- We tested the code on
+    - Desktop or VM using Ubuntu 20.10 (with tools installed of version Ubuntu 20.04 version since 20.10 is not officially supported) and Ubuntu 18.04 OS.
+    - Jetson Nano device.
+
+- We *did not* test on
+    - Windows OS
+    - MacOS
+
+- Using WSL (Windows Subsystem Linux) is NOT RECOMMENDED. You may run into unknown error messages.
```

Comment:
The error stated here https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-8.html#rel-8-4-0-EA
in the section "Known Issues". Could you help me add a link?